gulp/tasks/beep.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var fs = require( 'fs' );
+var gulp = require( 'gulp' );
+var path = require( 'path' );
+var Player = require( 'player' );
+
+var SONG_PATH = 'beep-build.mp3';
+
+/**
+ * Play an mp3 file using the Player npm module.
+ * @param {string} soundPath The path to a sound file.
+ */
+function _playSound( soundPath ) {
+  var player = new Player( soundPath );
+  player.play();
+
+  player.on( 'error', function() { // eslint-disable-line handle-callback-err, no-inline-comments, max-len
+    // Ignore playback errors because
+    // it's not critical if the beep doesn't play.
+  } );
+}
+
+gulp.task( 'beep', function() {
+
+  try {
+    var soundPath = path.join( __dirname, '../../' + SONG_PATH );
+    var stats = fs.lstatSync( soundPath ); // eslint-disable-line no-sync, no-inline-comments, max-len
+    if ( stats.isFile() ) {
+      _playSound( soundPath );
+    }
+  } catch ( e ) {
+    console.log( 'No build alert beep configured. Add a sound file as "' + // eslint-disable-line no-console, no-inline-comments, max-len
+                 SONG_PATH +
```

Comment:
It's aligned with the start of the string it's apart of, but doesn't need to be three lines. Fixed in https://github.com/cfpb/cfgov-refresh/commit/973601a2ba64e0d41ce32cea586a7dc948cf2dae